tests/integration.rs

```diff
@@ -457,3 +458,122 @@ fn recursive_multiple_root_workspaces() -> TestResult {
 
     Ok(())
 }
+
+/// This test follows the logic of the recursive multiple root test, however, instead of recursing it passes each workspace individually.
+
+#[test]
+fn multiple_paths() -> TestResult {
```

Comment:
This test still passes even without your changes. Not sure what's going wrong - maybe you're only asserting that the total size has decreased, instead of that both projects are being cleaned?

Reply:
Removing my change locally makes this fail so I'm also not sure what's going on 😅

Reply:
ah, I was testing this wrong, sorry - yes, I can also reproduce this failing without your change :) sorry for the noise.

Reply:
No worries!